### DISCUSSION TITLE
Is it possible to add the same Indicator to the candle_pane but with different calcParams ?

### DISCUSSION THREAD

**Discussion by zetzetto** [klinecharts/KLineChart#376] [Q&A]
Hi,
I wanted to add the same indicator, let's say the BOLL indicator twice but each with different parameters. But when I tried I can only see one at the main pane (candle_pane). Is there another way to add same indicator with different parameters to the main pane ( candle_pane ) ? Thank you.

**Comment by liihuu** [comment]
Currently, the chart does not allow technical indicators with the same name to be created in the same pane, but you can customize a technical indicator with different names, such as BOLL2, so that you can create BOLL and BOLL2 in the same pane.

**Reply by zetzetto** [reply]
Thank you, I will try to do that.
